pymc3/tests/test_distributions.py

```diff
@@ -1244,6 +1250,7 @@ def test_fun(value, mu, sigma):
         condition=(aesara.config.floatX == "float32"),
         reason="Fails on float32 due to numerical issues",
     )
+    # this test doesn't always result in the same outcome fail/success
```

Comment:
As mentioned above, that is probably due to subsampling of the test combinations, since the default is `n_samples=100`. I would either remove this comment or substitute it by a more informative one that explains why this is not a deterministic failure. 

Same for the other repeated comments

Reply:
Same as above. Once added `n_samples=-1` the result is consistent. Removing the comment

Reply:
Xfailed as expected in https://github.com/pymc-devs/pymc3/pull/4497/checks?check_run_id=2021551266